tests/core/test_ccall.cpp

```diff
@@ -36,11 +36,11 @@ int *pointer(int *in) {
 struct test_struct {
   int arg1, arg2, arg3;
 };
-static int* stackChecker = 0;
+static long* stackChecker = 0;
 __attribute__((noinline))
-int get_stack() { int i; return (int)&i; }
+long get_stack() { int i; return (long)&i; }
 int uses_stack(test_struct* t1) {
-  if (stackChecker == 0) stackChecker = (int*)malloc(sizeof(int));
+  if (stackChecker == 0) stackChecker = (long*)malloc(sizeof(long));
```

Comment:
I can't see why we use malloc at all here.. why not just have `static intptr_t stackChecker = 0` and avoid the malloc completely? (just store the value in memory)

Reply:
I was not trying to fix the actual tests, just make this 64-bit compatible. I can fix this also if you want, but it would not make this PR a wasm32 NFC as intended.